src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  justify-content: left;
+  width: mainWidth($day-width, $calendar-padding);
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-padding;
+
+  @for $days from 1 through $day-in-month {
```

Comment:
Replace the variable $day-in-month with 31, as we need to create a loop for each day from 1 to 31.